pop/README.md

```diff
@@ -0,0 +1,119 @@
+# Proof of Personhood
+
+pop command is used by one of the two groups of users:
```

Comment:
is used by both you mean ? Otherwise which group is using `pop` if there is only one ?